init.rb

```diff
@@ -28,3 +28,5 @@
 
 Autoproj::PackageManagers::BundlerManager
     .configure_build_for('gem_build_flags', '--with-toplevel-init=success')
+
+Autoproj.config.set "syskit_use_bundles", false, true
```

Comment:
why this is needed?

Reply:
It's not needed _per se_, but it's our default configuration and in this particular build configuration, I think it's fine.